build.atpkg

```diff
@@ -16,57 +16,102 @@
 
 (package
   :name "atbuild"
-  :import ["atpkg/build.atpkg"]
+  :import-packages ["atpkg/build"]
   
+  :overlays {
+    :attools-info {
+      :name "attools"
+      :module-name "AnarchyTools"
+      :output-type "static-library"
+      :sources ["attools/src/**.swift"]
+    
+      ;; This is should be automatically resolved from the dependencies.
```

Comment:
I don't know what this comment means?

I'm pretty sure it isn't automatically resolved from the dependencies, and I don't believe it should be...